docs/source/examples/checks/vision/distribution/train_test_label_drift.ipynb

```diff
@@ -201,7 +201,7 @@
          "showlegend": false,
```

Comment:
<p>Remember to update this example to our new format. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/797/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Will do later, as discussed